### PR TITLE
Add Swift Instructions

### PR DESCRIPTION
**Pull request by Crisfole**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +52,34 @@ Crashes will be saved to disk when they occur, then reported to Rollbar the next
...
+### Swift
+
+Importing with Swift requires the additional step of adding the following lines to your Bridging-Header file:
+
+```c
+#import <SystemConfiguration/SystemConfiguration.h>
+#import <Rollbar/Rollbar.h>
+```
+
+If you have no Bridging Header file, the easiest way to correctly configure it is to add an objective-c (`.m`) file. When you do so, XCode will prompt you to create a briding header file, and will configure your build environment to automatically include those headers in all your Swift files. After creating the Briging-Header file, you can delete the objective-c file.
```
Comment (brianr) [inline README.md]: typo in `Briging-Header` in last sentence

Comment (brianr) [inline README.md]: Is it enough to add an empty `.m` file? Maybe we should have the instructions say something like "add an empty file called `dummy.m`"

File: README.md [hunk 2]
```diff
@@ -48,6 +52,34 @@ Crashes will be saved to disk when they occur, then reported to Rollbar the next
...
+    let config: RollbarConfiguration = RollbarConfiguration()
+    config.environment = "production"
+
+    Rollbar.initWithAccessToken("YOUR ACCESS TOKEN", configuration: config)
+
+    return true
+}
+```
+
+See the last four commits in https://github.com/Crisfole/SwiftWeather for a demo of how to integrate Rollbar into an existing Swift project.
```
Comment (brianr) [inline README.md]: This comment is going to be out of date if that repo ever changes. Maybe you could link to the specific range instead? https://github.com/Crisfole/SwiftWeather/compare/18580ce...e7d80e1

File: README.md (hunk 1 above)
Comment (Crisfole) [inline README.md]: Good catch, and yes an empty `.m` file works.

File: README.md (hunk 2 above)
Comment (Crisfole) [inline README.md]: Good call. Although "that repo" is mine :)

**Comment by Crisfole** [comment]
@brianr  Fixed.

**Pull request merged**